src/js/components/ChromecastButton.js

```diff
@@ -63,7 +63,7 @@ ChromecastButton = {
     */
    buildCSSClass: function() {
       return 'vjs-chromecast-button ' + (this._isChromecastConnected ? 'vjs-chromecast-casting-state ' : '') +
-         this.constructor.super_.prototype.buildCSSClass();
+         (this.options_.addCastLabelToButton ? 'vjs-chromecast-button-lg ' : '') + this.constructor.super_.prototype.buildCSSClass();
```

Comment:
```suggestion
         return 'vjs-chromecast-button ' + 
            (this._isChromecastConnected ? 'vjs-chromecast-casting-state ' : '') +
            (this.options_.addCastLabelToButton ? 'vjs-chromecast-button-lg ' : '') + 
            this.constructor.super_.prototype.buildCSSClass();
```

Could we lay this function out like this for readability?